README.md

```diff
@@ -13,7 +13,7 @@ use minio::s3::http::BaseUrl;
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
-    let base_url = BaseUrl::from_string("https://play.min.io".to_string()).unwrap();
+    let base_url = "https://play.min.io".parse::<BaseUrl>()?;
```

Comment:
Can we embed `./examples/file-uploader.rs` here? If not possible, we could give a link to `file-uploader.rs` instead of duplicating content of the file here.  We could easily forget to update both the content.

Reply:
Removed it. There is no way to embed a live version of this code in the readme afaik.